File: v1-monolitico/schemas/ubicacion.py
from utils.ma import ma
from model.ubicacion import Ubicacion

class UbicacionSchema(ma.Schema):
    class Meta:
        model = Ubicacion
        fields = (
            'ubigeo',
            'distrito',
            'provincia',
            'y',
            'x',
        )

ubicacion_schema = UbicacionSchema()
ubicaciones_schema = UbicacionSchema(many=True)

File: v2-microservicios/sisvita-app/auth-server/src/main/java/sm/dsw/sisvita/auth/model/Ubicacion.java
package sm.dsw.sisvita.auth.model;

import jakarta.persistence.*;
import lombok.*;
import java.util.Date;

@Entity
@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Table(name = "ubicacion")

public class Ubicacion {
    @Id
    private Long ubigeo;

    @Column(name = "provincia")
    private String provincia;

    @Column(name = "distrito")
    private String distrito;

    @Column(name = "x")
    private double x;
    
    @Column(name = "y")
    private double y;
}
